app/src/main/java/com/vci/vectorcamapp/incomplete_session/presentation/IncompleteSessionScreen.kt

```diff
@@ -1,33 +1,62 @@
 package com.vci.vectorcamapp.incomplete_session.presentation
 
 import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.offset
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.unit.dp
 import com.vci.vectorcamapp.incomplete_session.presentation.components.IncompleteSessionCard
-import java.util.UUID
+import com.vci.vectorcamapp.incomplete_session.presentation.components.PageHeader
+import com.vci.vectorcamapp.ui.extensions.dimensions
+import com.vci.vectorcamapp.ui.theme.screenHeightFraction
 
 @Composable
 fun IncompleteSessionScreen(
     state: IncompleteSessionState,
-    onAction: (IncompleteSessionAction) -> Unit,
-    modifier: Modifier = Modifier
+    onAction: (IncompleteSessionAction) -> Unit
 ) {
```

Comment:
You should add the modifier back, and the first box should be modifier with the small "m" because the scaffold modifier from the NavGraph needs to be passed into the first component in the hierarchy.